src/button/example/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
If you rebase, this page can be deleted. You can hit your test page using the URL `http://your_local_server/_build/src/common/example/?module=Button`.